tests/bluetooth/tester/src/gap.c

```diff
@@ -738,6 +738,47 @@ static void auth_cancel(struct bt_conn *conn)
 	/* TODO */
 }
 
+struct bond_info_args {
+	const bt_addr_le_t *addr;
+	bool bond_found;
+};
+
+static void bond_info(const struct bt_bond_info *info, void *user_data)
+{
+	struct bond_info_args *args = user_data;
+
+	if (bt_addr_le_cmp(&info->addr, args->addr) == 0) {
+		args->bond_found = true;
+	}
+}
+
+enum bt_security_err auth_pairing_accept(struct bt_conn *conn,
+					 const struct bt_conn_pairing_feat *const feat)
+{
+	struct gap_bond_lost_ev ev;
+	const bt_addr_le_t *addr = bt_conn_get_dst(conn);
+	struct bond_info_args args = {
+		.addr = addr,
+		.bond_found = false,
+	};
+
+	bt_foreach_bond(BT_ID_DEFAULT, bond_info, &args);
+
+	if (!args.bond_found) {
+		return BT_SECURITY_ERR_SUCCESS;
+	}
+
+	LOG_DBG("bond lost");
```

Comment:
Not fully knowing the test case, it can be slightly confusing that a pairing accept callback logs "bond lost". Could you add a comment elaborating this in this callback?